client/src/Components/Layouts/CreateSession/index.js

```diff
@@ -0,0 +1,138 @@
+import React, { Component } from "react";
+import axios from "axios";
+import moment from "moment";
+import swal from "sweetalert";
+import ReactRouterPropTypes from "react-router-prop-types";
+
+import {
+  Button,
+  Form,
+  Contianer,
+  Input,
+  SelectComponent,
+  Date,
+  Heading,
+  Error,
+} from "./FormElements";
+import options from "./options";
+
+import "react-datepicker/dist/react-datepicker.css";
+
+class CreateSession extends Component {
+  static propTypes = {
+    history: ReactRouterPropTypes.history,
+  }
+
+  static defaultProps = {
+    history: null,
+  };
+
+  state = {
+    session: "",
+    startDate: null,
+    attendantsNumber: "",
+    err: false,
+  };
+
+  handleDateChange = (date) => {
+    this.setState({
+      startDate: date,
+    });
+  }
+
+  handleInputChange = ({ target: { value, name } }) => {
+    this.setState({
+      [name]: value,
+    });
+  }
+
+  handleSelectChange = (session) => {
+    this.setState({
+      session,
+    });
+  }
+
+  checkError = () => {
+    const { startDate, attendantsNumber, session } = this.state;
+    const isError = !(!!startDate && !!attendantsNumber && !!session);
```

Comment:
`!!` it returns `true` or `false` if the element is existed return true, otherwise it will return false 
[I found this on stackoverflow](https://stackoverflow.com/questions/784929/what-is-the-not-not-operator-in-javascript)

Reply:
I see! Thanks for the explanation